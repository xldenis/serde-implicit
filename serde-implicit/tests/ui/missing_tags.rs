#[derive(serde_implicit_proc::Deserialize)]
enum OopsTag {
    MissingTag {
        field1: String,
        field2: bool,
        value: u32,
    },
    SingleTagged {
        #[serde_implicit(tag)]
        only_tag: u32,
        value: String,
    },
}

fn main() {}
