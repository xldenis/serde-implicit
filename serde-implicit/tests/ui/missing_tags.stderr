error: missing `#[tag]`
 --> tests/ui/missing_tags.rs:3:5
  |
3 | /     MissingTag {
4 | |         field1: String,
5 | |         field2: bool,
6 | |         value: u32,
7 | |     },
  | |_____^
